metamusic/clients/musicbrainz.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/env python3
+"""
+Author : Salvador <salvador.estrella.ortiz@gmail.com>
+Date   : 2022-09-30
+Purpose: Query Metadata from MusicBrainz API
+"""
+
+import argparse
+import requests
+from pprint import pprint
+from time import sleep
+
+
+# --------------------------------------------------
+def get_args():
+    """Get command-line arguments"""
+
+    parser = argparse.ArgumentParser(
+        description="Retrieve metadata from MusicBrainz API",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+
+    parser.add_argument(
+        "-a",
+        "--artist",
+        help="An artist to search for",
+        metavar="str",
+        type=str,
+        required=True,
+    )
+
+    return parser.parse_args()
+
+
+# --------------------------------------------------
+class MusicMetadata:
+    artist_endpoit = "http://musicbrainz.org/ws/2/artist/"
+    release_groups_endpoint = "http://musicbrainz.org/ws/2/release-group/"
+    releases_endpoint = "http://musicbrainz.org/ws/2/release/"
+
+    @classmethod
+    def _pick_artist(cls, artist_list):
+        for artist in artist_list:
+            if artist["score"] == 100:
+                return artist
```

Comment:
**suggestion:** Set a todo or a NotImplementedError where things are pending to be done. Unless we don't want to deal with no artist and 100 score that way.

```suggestion
        for artist in artist_list:
            if artist["score"] == 100:
                return artist
        raise NotImplementedError("There's no 100 score artist. Can't handle it right now.")
```